### ISSUE TITLE
StaffAPI Update Metadata

### ISSUE THREAD

**Issue by cacique-coder** [PlaceOS/drivers#176]
**Is your feature request related to a problem? Please describe.**

Staff Api should include the method update_metadata

**Describe the solution you'd like**

Just having the method update_metadata

**Additional context**

I am pushing a new PR #175 

**Problems**

1. It is not compiling, the error is this:
  ```
  details : JSON::Any | Hash | NamedTuple | Array,                           
  Error: instantiating 'JSON::Any.class#|(Hash(K, V).class)'
  ```
2. Typing is coming from: https://github.com/PlaceOS/crystal-client/blob/master/src/placeos/api_wrapper/metadata.cr#L19
3. We have a custom Driver including the method needed. But when description is nil, we are receiving an`API:Error`

**Comment by stakach** [comment]
There is a new function on staff-api now
`def write_metadata(id : String, key : String, payload : JSON::Any, description : String? = nil)`

**Comment by caspiano** [comment]
Specifically closed by 2f82b6d542465591d0a965dca63a33f7d23239f9
